### PR TITLE
Add content/pt/docs/concepts/security/overview.md in pt language

### PR DESCRIPTION
**Pull request by krol3**
Add concepts/security/overview in Portuguese language. 
Related to issue #13939

### CODE REVIEW

File: content/pt/docs/concepts/security/overview.md [hunk 1]
```diff
@@ -0,0 +1,153 @@
...
+------------------------------ | --------------------- |
+Autorização RBAC (acesso à API Kubernetes) | https://kubernetes.io/docs/reference/access-authn-authz/rbac/
+Autenticação | https://kubernetes.io/docs/concepts/security/controlling-access/
+Gerenciamento de segredos na aplicação (e encriptando-os no etcd em repouso) | https://kubernetes.io/docs/concepts/configuration/secret/ <br> https://kubernetes.io/docs/tasks/administer-cluster/encrypt-data/
+Políticas de segurança do Pod | https://kubernetes.io/docs/concepts/policy/pod-security-policy/
+Qualidade de serviço (e gerenciamento de recursos de cluster) | https://kubernetes.io/docs/tasks/configure-pod-container/quality-service-pod/
+Políticas de Network | https://kubernetes.io/docs/concepts/services-networking/network-policies/
+TLS para Kubernetes Ingress | https://kubernetes.io/docs/concepts/services-networking/ingress/#tls
+
+## Container
```
Comment (jailton) [inline content/pt/docs/concepts/security/overview.md:110]: This section still in English.

File: content/pt/docs/concepts/security/overview.md [hunk 2]
```diff
@@ -0,0 +1,153 @@
...
+
+## {{% heading "whatsnext" %}} 
+
+Saiba mais sobre os tópicos de segurança do Kubernetes:
+
+* [Padrões de segurança do Pod](/docs/concepts/security/pod-security-standards/)
+* [Network policies para Pods](/docs/concepts/services-networking/network-policies/)
+* [Controle de acesso à API Kubernetes](/docs/concepts/security/controlling-access)
+* [Protegendo seu cluster](/docs/tasks/administer-cluster/securing-a-cluster/)
+* [Data encryption in transit](/docs/tasks/tls/managing-tls-in-a-cluster/) for the control plane
```
Comment (jailton) [inline content/pt/docs/concepts/security/overview.md:150]: If there is no Portuguese version of a page, you can translate the title and maintain the original link.

**Review by jailton — CHANGES_REQUESTED** [review]

**Comment by jailton** [comment]
Preview: https://deploy-preview-26719--kubernetes-io-master-staging.netlify.app/pt/docs/concepts/security/overview/

File: content/pt/docs/concepts/security/overview.md [hunk 3]
```diff
@@ -0,0 +1,153 @@
...
+Acesso do Kubernetes à API do provedor de Cloud | Cada provedor de nuvem precisa conceder um conjunto diferente de permissões para o control plane e nós do Kubernetes. É melhor fornecer ao cluster permissão de acesso ao provedor de nuvem que segue o [princípio do menor privilégio](https://en.wikipedia.org/wiki/Principle_of_least_privilege) para os recursos que ele precisa administrar. A [documentação do Kops](https://github.com/kubernetes/kops/blob/master/docs/iam_roles.md#iam-roles) fornece informações sobre as políticas e roles do IAM.
+Acesso ao etcd | O acesso ao etcd (o armazenamento de dados do Kubernetes) deve ser limitado apenas ao control plane. Dependendo de sua configuração, você deve tentar usar etcd sobre TLS. Mais informações podem ser encontradas na [documentação do etcd](https://github.com/etcd-io/etcd/tree/master/Documentation).
+Encriptação etcd | Sempre que possível, é uma boa prática encriptar todas as unidades em repouso, mas como o etcd mantém o estado de todo o cluster (incluindo os Secrets), seu disco deve ser criptografado em repouso.
```
Comment (jailton) [inline content/pt/docs/concepts/security/overview.md:74]: Sugestions: 
- **unidades em repouso** -> unidades de armazenamento
- **seu disco deve ser criptografado em repouso** -> seu disco deve ser criptografado

**Review by jailton — CHANGES_REQUESTED** [review]

File: content/pt/docs/concepts/security/overview.md (hunk 1 above)
Comment (krol3) [reply content/pt/docs/concepts/security/overview.md:110]: The word and k8s component called "Ingress", could I use it?

File: content/pt/docs/concepts/security/overview.md (hunk 2 above)
Comment (jcjesus) [reply content/pt/docs/concepts/security/overview.md]: Hello Carol, I have been thinking about other links, which can also be translated, even if it is not possible to translate the page now.
What did you think @jailton?

File: content/pt/docs/concepts/security/overview.md [hunk 4]
```diff
@@ -0,0 +1,153 @@
...
+Se você deseja proteger seu cluster de acesso acidental ou malicioso e adotar
+boas práticas de informação, leia e siga os conselhos sobre
+[protegendo seu cluster](/docs/tasks/administer-cluster/securing-a-cluster/).
+
+### Componentes no cluster (sua aplicação) {#cluster-applications}
+
+Dependendo da superfície de ataque de sua aplicação, você pode querer se concentrar em
+aspectos específicos de segurança. Por exemplo: se você estiver executando um serviço (Serviço A) que é crítico
+numa cadeia de outros recursos e outra carga de trabalho separada (Serviço B) que é
+vulnerável a um ataque de exaustão de recursos e, por consequencia, o risco de comprometer o Serviço A
```
Comment (devlware) [inline content/pt/docs/concepts/security/overview.md:96]:
```suggestion
vulnerável a um ataque de exaustão de recursos e, por consequência, o risco de comprometer o Serviço A
```

File: content/pt/docs/concepts/security/overview.md (hunk 1 above)
Comment (devlware) [reply content/pt/docs/concepts/security/overview.md:110]: I believe that you can keep _ingress_ @krol3.

**Review by devlware — COMMENTED** [review]
Thanks for taking the time to help the project @krol3. I left some suggestions.

**Pull request merged**